### PR TITLE
6 get api articles articleid comments

### PR DESCRIPTION
**Pull request by micknice**
used for loop for descending date test as cant get jest-sort to fail

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ xdescribe('GET /api/articles', () => {
         })
     })
 })
-describe('get /api/articles/:article_id', ()=> {
+xdescribe('get /api/articles/:article_id', ()=> {
```
Comment (mallionaire) [inline __tests__/app.test.js]: Don't "x" out any tests

File: __tests__/app.test.js [hunk 2]
```diff
@@ -87,10 +87,42 @@ describe('get /api/articles/:article_id', ()=> {
...
+                    article_id: expect.any(Number)
+                })
+            })
+        })
+    })
+    test('comments object array is ordered from most recent', () => {
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(response => {
```
Comment (mallionaire) [inline __tests__/app.test.js:117]: You may find an npm package called
[jest-sorted](https://www.npmjs.com/package/jest-sorted)   to be helpful here.

File: controllers/error-handling-controller.js [hunk 3]
```diff
@@ -1,16 +1,13 @@
...
 const handleServerErrors = (err, req, res, next) => {
-    console.log(err)
+    console.log('handle server error', err)
     res.status(500).send({msg: 'Internal Server Error'})
 }
 
-
 const handlePSQL400Error = (err, req, res, next) => {
-    console.log(err)
+    console.log('handle psql 400 error', err)
```
Comment (mallionaire) [inline controllers/error-handling-controller.js]: Remove console.logs

File: models/model.js [hunk 4]
```diff
@@ -43,8 +43,32 @@ const fetchArticleById = (article_id) => {
...
+        `
+        SELECT comments.comment_id, comments.votes, comments.created_at, comments.author, comments.body, comments.article_id
+        FROM comments
+        WHERE comments.article_id = $1
+        ORDER BY comments.created_at DESC
+        `, [article_id]
+    )
+    .then(result => {
+        const comments = result.rows;
+        console.log('model comments.length', comments.length)
```
Comment (mallionaire) [inline models/model.js]: remove console.logs

**Review by mallionaire — CHANGES_REQUESTED** [review]
I'd like you to take a look at an npm package that will help make your tests much neater. 
more details in a comment below. 

Including console.logs is not cool at this stage in the week - its a sign that your code is not finished, and therefore not ready for a pull request. Make sure you don't include any unfinished code going forwards please :smile: 

Once you've made these changes pop in another pr and we'll take a look.

File: __tests__/app.test.js (hunk 2 above)
Comment (micknice) [reply __tests__/app.test.js:117]: hi Haz yeah Modou showed me yesterday but i cant get it to fail so i just popped  the old test back in for the sake of expediency until i can get some help with it

**Review by HannahBurks — APPROVED** [review]
Great work Mick - feel free to merge. If you are struggling with Jest sorted, I would strongly recommend looking at your package.json once you have installed it, this may be the issue as it has been for a few people. You want to ensure it looks like this:
devdependecies should have this underneath
  "jest-sorted": "^1.0.12",

and the jest should look like this :
  "jest": {
    "setupFilesAfterEnv": [
      "jest-extended",
      "jest-sorted"

**Pull request merged**